### PR TITLE
[influxdb] Add docs for usage of self-signed certs

### PR DESCRIPTION
**Pull request by florian-h05**
See
https://community.openhab.org/t/influxdb-persistence-doesnt-work-with-s sl/35409/10?u=florian-h05.

Signed-off-by: Florian Hotze <florianh_dev@icloud.com>

### CODE REVIEW

File: bundles/org.openhab.persistence.influxdb/README.md [hunk 1]
```diff
@@ -106,3 +106,13 @@ Besides the metadata tags, there are additional configuration parameters to acti
...
 | addLabelTag    | false   | no       | Should the item label be included as tag "label"? If no label is set, "n/a" is used.                 |
+
+### Connect to InfluxDB via TLS
+
+InfluxDB supports TLS encryption to secure the communication with clients.
+
+If you use a self-signed certificate for your InfluxDB instance (which is very likely), you need to add the certificate itself or your internal CA's certificate to the Java keystore:
+
+1. Find your JVM's path with `ls -all /usr/bin/java`, e.g. `/opt/java/zulu17.38.21-ca-jdk17.0.5-linux_aarch32hf/bin/java`. You may need to follow some symlinks, use `ls -all` again.
+2. Go to the `lib/security` directory of your JVM, e.g. `cd /opt/java/zulu17.38.21-ca-jdk17.0.5-linux_aarch32hf/lib/security`.
```
Comment (jlaur) [inline bundles/org.openhab.persistence.influxdb/README.md:117]:
```suggestion
1. Go to the `lib/security` directory of your JVM, e.g. `cd /opt/java/zulu17.38.21-ca-jdk17.0.5-linux_aarch32hf/lib/security`.
```

File: bundles/org.openhab.persistence.influxdb/README.md [hunk 2]
```diff
@@ -106,3 +106,13 @@ Besides the metadata tags, there are additional configuration parameters to acti
...
+3. Add the certificate to the JVM's keystore: `sudo keytool -importcert -file <path-to-certfile> -cacerts -keypass changeit -storepass changeit -alias <alias-for-cert>`.
```
Comment (jlaur) [inline bundles/org.openhab.persistence.influxdb/README.md:118]:
```suggestion
1. Add the certificate to the JVM's keystore: `sudo keytool -importcert -file <path-to-certfile> -cacerts -keypass changeit -storepass changeit -alias <alias-for-cert>`.
```

**Review by jlaur — CHANGES_REQUESTED** [review]
Thanks, only two minor comments for auto-numbering list.

**Comment by florian-h05** [comment]
@jlaur Sorry, I don‘t understand your suggested changes.
Do you want to reorder the list items?

**Comment by jlaur** [comment]
> Sorry, I don‘t understand your suggested changes.
> Do you want to reorder the list items?

Just to number the list with only 1's:
1. One.
1. Two.
1. Three

Source:
```
1. One.
1. Two.
1. Three
```

I saw @Confectrician do this recently with the advantage that the list is then auto-numbered and won't break if lines are added or removed.

**Comment by florian-h05** [comment]
Okay, makes sense. I wasn‘t aware that this is possible, learned something new :-)

I‘ve addressed your review.

**Review by jlaur — APPROVED** [review]
Thanks!

**Pull request merged**